### ISSUE TITLE
Through2 needs callback 

### ISSUE THREAD

**Issue by afanasy** [GoogleCloudPlatform/gcloud-node#963]
https://googlecloudplatform.github.io/gcloud-node/#/docs/v0.25.1/bigquery/job

In this example, through2.obj needs the callback call?
```js
var through2 = require('through2');
var fs = require('fs');

job.getQueryResults()
  .pipe(through2.obj(function (row, enc, next) {
    this.push(JSON.stringify(row) + '\n');
//next(); is missing from docs
  }))
  .pipe(fs.createWriteStream('./test/testdata/testfile.json'));
```

**Comment by stephenplusplus** [comment]
Good catch! Any chance you're free to do a PR?

**Comment by stephenplusplus** [comment]
Fixed by #964
